src/node_crypto_bio.h

```diff
@@ -96,6 +96,20 @@ class NodeBIO : public MemoryRetainer {
     return length_;
   }
 
+  // Provide a hint about the size of the next pending set of writes. TLS
+  // writes records of a maximum length of 16k of data plus a 5-byte header,
+  // a MAC (up to 20 bytes for SSLv3, TLS 1.0, TLS 1.1, and up to 32 bytes
+  // for TLS 1.2), and padding if a block cipher is used.  If there is a
+  // large write this will result in potentially many buffers being
+  // allocated and gc'ed which can cause long pauses.   By providing a
+  // guess about the amount of buffer space that will be needed in the
+  // next allocation this overhead is removed.
+  inline void set_allocate_tls_hint(size_t size) {
+    if (size >= 16 * 1024) {
+      allocate_hint_ = ((size / (1024 * 16) + 1) * ((1026 * 16) + 5 + 32));
```

Comment:
Typo?
```suggestion
      allocate_hint_ = ((size / (1024 * 16) + 1) * ((1024 * 16) + 5 + 32));
```
I suggest factoring out the constant into, er, a constant for readability. E.g:
```c++
constexpr size_t kThreshold = 16 * 1024;
if (size >= kThreshold) {
  allocate_hint_ = (size / kThreshold + 1) * (kThreshold + 5 + 32);
}
```
That said, the calculation doesn't look Obviously Correct to me. It yields 32,842 for all `16384 <= size <= 32767`.

Reply:
Yes thats a typo.